src/impl_x86__base_implementation.inl

```diff
@@ -667,6 +667,9 @@ X86Microarchitecture GetX86Microarchitecture(const X86Info* info) {
       case CPUID(0x12, 0x01):
         // https://www.amd.com/system/files/TechDocs/44739_12h_Rev_Gd.pdf
         return AMD_K12;
+      case CPUID(0x12, 0x00):
+        // https://www.amd.com/system/files/TechDocs/44739_12h_Rev_Gd.pdf
```

Comment:
I don't see LLANO by the link that you provided, so change it to https://en.wikichip.org/wiki/amd/cpuid, since there is a definition cpuid of LLANO

Reply:
+1 this link will be cool:
https://en.wikichip.org/wiki/amd/cpuid#Family_18_.2812h.29

Reply:
No more needed as I removed LLANO identifier. It has been merged with K12